tiny-event-sourcing-spring-app/src/main/kotlin/ru.quipy/demo/UserPaymentsViewDomain.kt

```diff
@@ -0,0 +1,79 @@
+package ru.quipy.demo
+
+import org.slf4j.*
+import org.springframework.data.annotation.Id
+import org.springframework.data.mongodb.core.mapping.Document
+import org.springframework.data.mongodb.repository.MongoRepository
+import org.springframework.stereotype.Repository
+import org.springframework.stereotype.Service
+import ru.quipy.demo.*
+import ru.quipy.domain.Unique
+import ru.quipy.streams.AggregateSubscriptionsManager
+import java.util.*
+import javax.annotation.PostConstruct
+
+
+class UserPaymentsViewDomain {
```

Comment:
comments. What is view in general, what is domain. Mention that it can be located in different service

Reply:
I've written about the views, but don't know what to write about domain, and don't understand what does it mean to be located in different service